packages/platform-sdk/test/integration-tests/sdk-v3/middlewares.test.ts

```diff
@@ -115,23 +115,26 @@ describe('Concurrent Modification Middleware', () => {
       }
     )
 
-    const productUpdateResponse = await apiRootV3
-      .products()
-      .withId({ ID: product.id })
-      .post({
-        body: {
-          version: product.version - 2,
-          actions: [
-            {
-              action: 'changeName',
-              name: { en: 'test-name' + new Date().getTime() },
-            },
-          ],
-        },
-      })
-      .execute()
-
-    expect(productUpdateResponse.statusCode).toBe(200)
+    try {
```

Comment:
Why did you add try/catch? The test should actually fail if there is an exception.